new-module/README.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Here it is more of an article, like stuff from the whitepaper. But essentially, what we want people to see when they come to the doc is understanding of the new module (and not paragraphs on why we made it). We could have a separate page for it, but it should not be the main page

Reply:
So you think I could just leave the vault directory README as main README?

Reply:
We need a small intro about it that rapidly describes how it works and the logic of the new module (like we do for the Core Module)

Reply:
So we're not changing this?

Reply:
We need to write at the present but put a disclaimer saying that borrowing module is not live yet